documentation/docs/01-routing.md

```diff
@@ -134,12 +129,18 @@ A route can have multiple dynamic parameters, for example `src/routes/[category]
 }
 ```
 
-Finally, you can use a subset of regular expression syntax to control whether routes match or not:
+#### Fallthrough routes
 
-```bash
-# matches /2021/04/25 but not /a/b/c or /1/2/3
-src/routes/[year(\d{4})]/[month(\d{2})]/[day(\d{2})].svelte
+Finally, if you have multiple routes that match a given path, SvelteKit will try each of them until one responds. For example if you have these routes...
+
+```
+src/routes/foo-[bar].svelte
+src/routes/[baz].svelte
+src/routes/[qux].svelte
 ```
 
-Because of technical limitations, the following characters cannot be used: `/`, `\`, `?`, `:`, `(` and `)`.
+...and you navigate to `/foo-xyz`, then SvelteKit will first try `foo-[bar].svelte` because it is the best match, then will try `[baz].svelte` and `[qux].svelte` (which are also valid matches for `/foo-xyz`, but less specific) in alphabetical order. The first page that returns something from [`load`](#loading), or has no `load` function, will handle the request.
+
+Similarly, multiple endpoints that match a given path will 'fall through' until one of them responds with something.
```

Comment:
Do you think it's worth noting here any of the stuff about how endpoints and pages might be interwoven in the fallback priority list?

Reply:
yes, good point. done